alert-policies/ext-ambassador/no-active-connections.yml

```
@@ -0,0 +1,62 @@
+---
+# Name of the alert
+name: Ambassador | No active connections for 5m
+
+# Description and details
+description: |+
+  This metric indicates whether a connection is active.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+  query: "SELECT latest(envoy_http_downstream_cx_http1_active) FROM Metric where endpoint ='ambassador-admin'  FACET instance"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE 
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: EQUAL
```

```suggestion
    operator: EQUALS
```
`operator` must be one of: ABOVE, BELOW, EQUALS